tracer/src/Datadog.Trace/ClrProfiler/CallTarget/Handlers/Continuations/ContinuationGenerator.cs

```diff
@@ -34,5 +39,27 @@ protected static TTo FromTReturn<TTo>(TReturn returnValue)
             return ContinuationsHelper.Convert<TReturn, TTo>(returnValue);
 #endif
         }
+
+        internal abstract class ContinuationResolver
+        {
+            public abstract TReturn SetContinuation(TTarget instance, TReturn returnValue, Exception exception, in CallTargetState state);
+        }
+
+        internal class NullContinuationResolver : ContinuationResolver
```

Comment:
naming nit: Changing `Null` to `NoOp`

Reply:
done in [79f1332](https://github.com/DataDog/dd-trace-dotnet/pull/3555/commits/79f1332071c1b502de0d0a058d9bd942081a5841)